extensions/scientific/README.md

```diff
@@ -1,10 +1,10 @@
-# STAC Scientific Extension Spec (`sci`)
+# Scientific Extension Specification (`sci`)
 
 **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
 
 This document explains the fields of the STAC Scientific Extension to a STAC Catalog. Scientific
-metadata is considered to be data that indicate from which publication a dataset originates and how
-the dataset itself should be cited or referenced. Overall, it helps to increase reproducibility and
+metadata is considered to be data that indicate from which publication a collection (dataset) originates and how
+the collection itself should be cited or referenced. Overall, it helps to increase reproducibility and
```

Comment:
Update if sci is changed to Item scope.

Reply:
Updated with aab3f4073c342746ace37982b35ac5c41017cfec